### PR TITLE
Artist overview formatting

### PR DESCRIPTION
**Pull request by localastronaut**

### CODE REVIEW

File: templates/artist/includes/artist_detail_countdown.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
                 <span class="goal-label">Funded</span>
             </div>
             <div class="goal-stat">
-                <span class="goal-number">${{ campaign.amount_raised|intcomma }}</span>
-                <span class="goal-label">Raised</span>
+                <span class="goal-number">${{ campaign.amount|intcomma }}</span>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_countdown.html]: We should probably revert this change, since the `artist_detail_countdown` is now only used in the past campaigns section.

File: templates/artist/includes/artist_detail_overview.html [hunk 2]
```diff
@@ -4,140 +4,125 @@
...
-    {% if campaign %}
-        <div class="break">
-            <hr>
+    {% thumbnail artist.photo.img "500x500" crop="center" as thumb %}
+        <div class="photo-left">
+            <img class="dropshadow" src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
         </div>
+    {% endthumbnail %}
+    {% with campaign=campaign %}
+    <div class="buy-right">
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: nit: Everything inside the `{% with %}` tag should be indented.

File: templates/artist/includes/artist_detail_overview.html [hunk 3]
```diff
@@ -4,140 +4,125 @@
...
+                    <button id="add-shares" class="secondary hollow button" type="button">+</button>
+                </div>
+                <p>NUMBER OF SHARES</p> 
+                <div class="total-cost">
+                    <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)">
+                        Total <a id="shares-price"></a>
+                    </span>
+                </div>
+            {% else %}
+                <h4> to invest in this project</h4>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: nit: Remove space before "to"

File: templates/artist/includes/artist_detail_overview.html [hunk 4]
```diff
@@ -4,140 +4,125 @@
...
+                <button class="login-box signup" data-open="login-modal">Login</button>
+                <button class="login-box signup" >Sign up</button>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: Did we want to change Sign up to be a modal just like Login? Or should this button link to the register page?

File: templates/artist/includes/artist_detail_overview.html [hunk 5]
```diff
@@ -4,140 +4,125 @@
...
+            {% endif %}
+    </div>
+    {% endwith %}
+
+        <!-- Music -->
+        {% if artist.soundcloudplaylist_set.exists %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: This `{% if %}` block should just surround the Soundcloud playlist. Otherwise the genres and artist location will not show if they don't have any music to preview.

File: templates/artist/includes/artist_detail_overview.html [hunk 6]
```diff
@@ -4,140 +4,125 @@
...
                 <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
             {% endfor %}
         </div>
-    {% endif %}
+        {% endif %}
 
-    {% if campaign %}
-        {% if campaign.expense_set.all %}
+        <!-- Artist Bio -->
+        <h4>Bio</h4>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: The Bio header should be inside the `{% if artist.bio %}` block.

File: templates/artist/includes/artist_detail_overview.html [hunk 7]
```diff
@@ -4,140 +4,125 @@
...
+        {% if artist.bio %}
+            <div class="artist-detail-padding">
+                {{ artist.bio.bio|markdown }}
+            </div>
+        {% endif %}
+        {% if campaign %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: The `{% if campaign %}` block should come before the "WHAT DO YOU GET FOR INVESTING?" header.

File: templates/artist/includes/artist_detail_overview.html [hunk 8]
```diff
@@ -4,140 +4,125 @@
...
+                    <li>{{ percentage_breakdown.name }}: {{ percentage_breakdown.percentage|notrail_floatformat:2 }}%</li>
+                {% endfor %}
+                <li>
+                    {% if user_investor %}Other Investors:{% else %}Investors:{% endif %}
+                    {{ fans_percentage|notrail_floatformat:2 }}%
+                </li>
+                {% endif %}
+            </ul>
+            <h4 class="investing">You'll also get</h4>
+            <p class="investing-copy">To hear the music before anyone else</p>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: Should this be bullet points? Either way I think `<ul>` would be more appropriate than `<p>`s.

File: templates/artist/includes/artist_detail_overview.html [hunk 9]
```diff
@@ -4,140 +4,125 @@
...
+            <p class="investing-copy">To download the project as soon as it's ready</p>
+            <p class="investing-copy">Exclusive updates from the artist</p>
+            <div class="break">
+                <hr>
+            </div>
+            <!-- What is the money being used for? -->
+            {% if campaign %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: We can merge this if statement, and the one below with `{% if campaign and campaign.project.generated_revenue %}`. Once we change that, we should remove the inner `{% endif %}`.

File: templates/extra/privacy.html [hunk 10]
```diff
@@ -41,7 +41,7 @@ <h1 class="privacy-title">Privacy</h1>
         <p>Only persons age 18 or older have permission to access our Service. Our Service does not address anyone under the age of 13 ("Children").</p>
         <p>We do not knowingly collect personally identifiable information from children under 13. If you are a parent or guardian and you learn that your Children have provided us with Personal Information, please contact us. If we become aware that we have collected Personal Information from a children under age 13 without verification of parental consent, we take steps to remove that information from our servers.</p>
         <h5>Changes To This Privacy Policy</h5>
-        <p>This Privacy Policy is effective as of September 17, 2015 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.</p>
+        <p>This Privacy Policy is effective as of July 10, 2016 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.</p>
```
Comment (RevolutionTech) [inline templates/extra/privacy.html]: Update date to July 11th

File: templates/extra/privacy.html [hunk 11]
```diff
@@ -17,7 +17,7 @@ <h1 class="privacy-title">Privacy</h1>
...
-        <p>In addition, we may use third party services such as Google Analytics that collect, monitor and analyze this type of information in order to increase our Service's functionality. These third party service providers have their own privacy policies addressing how they use such information.</p>
+        <p>In addition, we may use third party services such as Google Analytics that collect, monitor and analyze this type of information in order to increase our Service's functionality. We may collect information from such visitor regarding his use of that Website, such as pages visited, links clicked, non-sensitive text entered, and mouse movements, as well as information more commonly collected such as his IP address, referring URL, browser, operating system, cookie information, and Internet Service Provider ("Visitors' Information”). These third party service providers have their own privacy policies addressing how they use such information.</p>
```
Comment (RevolutionTech) [inline templates/extra/privacy.html]: Would rephrase

> We may collect information from such visitor regarding his use of that Website, such as pages visited, links clicked, non-sensitive text entered, and mouse movements, as well as information more commonly collected such as his IP address, referring URL, browser, operating system, cookie information, and Internet Service Provider ("Visitors' Information”).

since our case is slightly different from Jaco's. You could remove the section completely and then edit the above line:

> We collect information that your browser sends whenever you visit our Service ("Log Data"). This Log Data may include information such as your computer's Internet Protocol ("IP") address, browser type, browser version, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages and other statistics.

to:

> We collect information that your browser sends whenever you visit our Service ("Log Data"). This Log Data may include information such as your computer's Internet Protocol ("IP") address, Internet Service Provider, referring URL, browser type, browser version, operating system, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages, links clicked, non-sensitive text entered, mouse movements, cookie information, and other statistics.

File: templates/artist/includes/artist_detail_overview.html (hunk 4 above)
Comment (localastronaut) [inline templates/artist/includes/artist_detail_overview.html]: Yes! Would it be possible to make it a pop up module?

File: templates/artist/includes/artist_detail_countdown.html [hunk 12]
```diff
@@ -12,7 +12,7 @@
                 <span class="goal-label">Funded</span>
             </div>
             <div class="goal-stat">
-                <span class="goal-number">${{ campaign.amount_raised|intcomma }}</span>
+                <span class="goal-number">${{ amount_raised|intcomma }}</span>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_countdown.html]: This should be `{{ campaign.amount_raised|intcomma }}`.

File: templates/artist/includes/artist_detail_overview.html [hunk 13]
```diff
@@ -4,140 +4,125 @@
...
-    {% if artist.bio %}
-        <div class="break">
-            <hr>
-        </div>
-        <!-- Artist Bio -->
-        <h2>OUR STORY</h2>
-        <div class="artist-detail-padding">
-            {{ artist.bio.bio|markdown }}
+
+                {% if request.user.is_authenticated %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: nit: This section is indented one tab too much.

File: templates/artist/includes/artist_detail_overview.html [hunk 14]
```diff
@@ -4,140 +4,125 @@
...
+                    <button class="login-box signup" data-open="login-modal">Login</button>
+                    <button class="login-box signup" >Sign up</button>
+                {% endif %}
         </div>
-    {% endif %}
+    {% endwith %}
 
-    {% if artist.soundcloudplaylist_set.exists %}
+        <!-- Music -->
+        {% if artist.soundcloudplaylist_set.exists %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: This line should go before line 58

File: templates/artist/includes/artist_detail_overview.html [hunk 15]
```diff
@@ -4,140 +4,125 @@
...
+            <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
+        {% endwith %}
+        <h4>Location: {{ artist.location }}</h4>
         <div class="artist-detail-padding">
             {% for playlist in artist.soundcloudplaylist_set.all %}
                 <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
             {% endfor %}
         </div>
-    {% endif %}
+        {% endif %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: The endif should surround the SoundCloud stuff

File: templates/artist/includes/artist_detail_overview.html [hunk 16]
```diff
@@ -4,140 +4,125 @@
...
-        <div class="break">
-            <hr>
-        </div>
-        <!-- Socials -->
-        <h2>FOLLOW US!</h2>
-        {% for social in artist.social_set.all %}
-            <a href="{{ social.url }}"><i class="fa fa-{{ social.get_medium_display|lower }} fa-2x social-padding" aria-hidden="true"></i></a>
-        {% endfor %}
-    {% endif %}
+            <h4>Share this project</h4>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: This section should be unindented one tab.

File: templates/extra/privacy.html [hunk 17]
```diff
@@ -7,7 +7,7 @@ <h1 class="privacy-title">Privacy</h1>
     </div>
     <div class="extra-padding">
         <h5>PRIVACY POLICY</h5>
-        <h5>LAST UPDATED: SEPTEMBER 17, 2015</h5>
+        <h5>LAST UPDATED: July 11, 2016</h5>
```
Comment (RevolutionTech) [inline templates/extra/privacy.html]: To be consistent, I suppose we should have July in all caps.

File: templates/extra/privacy.html [hunk 18]
```diff
@@ -16,8 +16,8 @@ <h1 class="privacy-title">Privacy</h1>
...
+        <p>We collect information that your browser sends whenever you visit our Service ("Log Data"). This Log Data may include information such as your computer's Internet Protocol ("IP") address, Internet Service Provider, referring URL, browser type, browser version, operating system, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages, links clicked, non-sensitive text entered, mouse movements, cookie information, and other statistics.</p>
+        <p>In addition, we may use third party services such as Google Analytics that collect, monitor and analyze this type of information in order to increase our Service's functionality. We may collect information from such visitor regarding his use of that Website, such as pages visited, links clicked, non-sensitive text entered, and mouse movements, as well as information more commonly collected such as his IP address, referring URL, browser, operating system, cookie information, and Internet Service Provider ("Visitors' Information”). These third party service providers have their own privacy policies addressing how they use such information.</p>
```
Comment (RevolutionTech) [inline templates/extra/privacy.html]: This line should be reverted since we added the clauses needed in the line before.

File: templates/extra/privacy.html [hunk 19]
```diff
@@ -41,7 +41,7 @@ <h1 class="privacy-title">Privacy</h1>
         <p>Only persons age 18 or older have permission to access our Service. Our Service does not address anyone under the age of 13 ("Children").</p>
         <p>We do not knowingly collect personally identifiable information from children under 13. If you are a parent or guardian and you learn that your Children have provided us with Personal Information, please contact us. If we become aware that we have collected Personal Information from a children under age 13 without verification of parental consent, we take steps to remove that information from our servers.</p>
         <h5>Changes To This Privacy Policy</h5>
-        <p>This Privacy Policy is effective as of September 17, 2015 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.</p>
+        <p>This Privacy Policy is effective as of July 11, 2016 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.</p>
```
Comment (RevolutionTech) [inline templates/extra/privacy.html]: I hadn't read this line before, but actually I think we want to leave this line as it was, as the date referred to when it originally went into effect.

File: templates/artist/includes/artist_detail_overview.html [hunk 20]
```diff
@@ -4,140 +4,125 @@
...
-            <hr>
-        </div>
-        <h2>Buy Shares</h2>
-        {% if request.user.is_authenticated %}
-            <div class="invest-num-shares">
-                <button id="remove-shares" class="secondary hollow button disabled" type="button" disabled>-</button>
-                <input type="number" min="1" max="{{ campaign.num_shares_remaining }}" value="{{ campaign.default_num_shares }}" />
-                <button id="add-shares" class="secondary hollow button" type="button">+</button>
+    {% endthumbnail %}
+    {% with campaign=campaign %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: nit: Add newline above this line to separate these sections

File: templates/artist/includes/artist_detail_overview.html [hunk 21]
```diff
@@ -4,140 +4,125 @@
...
+            <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
+        {% endwith %}
+        <h4>Location: {{ artist.location }}</h4>
+        {% if artist.soundcloudplaylist_set.exists %}
         <div class="artist-detail-padding">
             {% for playlist in artist.soundcloudplaylist_set.all %}
                 <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
             {% endfor %}
+        {% endif %}
         </div>
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: This `</div>` should be inside the `{% endif %}`.

File: templates/artist/includes/artist_detail_overview.html [hunk 22]
```diff
@@ -4,140 +4,125 @@
...
+                <h4>to invest in this project</h4>
+                <button class="login-box signup" data-open="login-modal">Login</button>
+                <button class="login-box signup" >Sign up</button>
+            {% endif %}
         </div>
-    {% endif %}
+    {% endwith %}
 
-    {% if artist.soundcloudplaylist_set.exists %}
+        <!-- Music -->
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: Everything here can be unindented to line up with the `{% endwith %}`.

File: templates/artist/includes/artist_detail_overview.html [hunk 23]
```diff
@@ -4,140 +4,125 @@
...
             {% for playlist in artist.soundcloudplaylist_set.all %}
                 <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
             {% endfor %}
+        {% endif %}
         </div>
-    {% endif %}
 
-    {% if campaign %}
-        {% if campaign.expense_set.all %}
+        <!-- Artist Bio -->
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: This and the next line can be unindented.

File: templates/artist/includes/artist_detail_overview.html [hunk 24]
```diff
@@ -4,140 +4,125 @@
...
+        {% if artist.bio %}
+        <h4>Bio</h4>
+            <div class="artist-detail-padding">
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: The bio should be unindented once, as well.

File: templates/artist/includes/artist_detail_overview.html [hunk 25]
```diff
@@ -4,140 +4,125 @@
...
+                {{ artist.bio.bio|markdown }}
+            </div>
+        {% endif %}
+        {% for social in artist.social_set.all %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: nit: socials should be unindented.

File: templates/artist/includes/artist_detail_overview.html [hunk 26]
```diff
@@ -4,140 +4,125 @@
...
+            <a href="{{ social.url }}"><i class="fa fa-{{ social.get_medium_display|lower }} fa-2x social-padding" aria-hidden="true"></i></a>
+        {% endfor %}
             <div class="break">
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: Break should be unindented

File: templates/artist/includes/artist_detail_overview.html [hunk 27]
```diff
@@ -4,140 +4,125 @@
...
-        </div>
-        <ul>
-            {% for percentage_breakdown in campaign.project.artist_percentage %}
-                <li>{{ percentage_breakdown.name }}: {{ percentage_breakdown.percentage|notrail_floatformat:2 }}%</li>
-            {% endfor %}
-            <li>
-                {% if user_investor %}Other Investors:{% else %}Investors:{% endif %}
-                {{ fans_percentage|notrail_floatformat:2 }}%
-            </li>
+                {% endif %}
```
Comment (RevolutionTech) [inline templates/artist/includes/artist_detail_overview.html]: This `{% endif %}` seems misplaced.

File: static/css/home.css [hunk 28]
```diff
@@ -13,11 +13,13 @@
 
 h1.home-title {
     color: #ffffff;
+    text-shadow: 2px 2px 7px rgba(0, 0, 0, 0.7);
```
Comment (RevolutionTech) [inline static/css/home.css]: Remove drop shadows, since we now have a gradient
